documentation/configuration.markdown

```diff
@@ -195,3 +195,34 @@ a multi-master environment or using an external CA, you might want to disable
 the CA service on some nodes.
 
 
+## Java Secure Socket Extension (JSSE) configuration
+
```

Comment:
@nfagerlund FYI, we've added a note to the puppet server configuration documentation about how to re-enable SSLv3, if required.

Re-enabling SSLv3 is not advised, at all, due to POODLE.